packages/coin98-wallet/src/lib/coin98-wallet.ts

```diff
@@ -114,39 +114,14 @@ const Coin98Wallet: WalletBehaviourFactory<InjectedWallet> = async ({
       return getAccounts();
     },
 
-    async verifyOwner({ message }) {
+    async verifyOwner() {
       const account = getActiveAccount(store.getState());
 
       if (!account) {
         throw new Error("No active account");
       }
```

Comment:
If we're removing the other code then let's remove this too, and throw the below error immediately.